test/common/README.md

```diff
@@ -498,6 +498,40 @@ Returns the result of
 Returns the result of
 `fs.readFileSync(path.join(fixtures.fixturesDir, 'keys', arg), 'enc')`.
 
+## Internet Module
```

Comment:
Nit: Can you please add an entry for this new module to the table of contents near the start of this file?

Reply:
@Trott Of course!